### PR TITLE
Math-Magician: Full website

### PR DESCRIPTION
**Pull request by Kaiserabbas**
## Developed a full website with several pages.

- Created a website consisting of 3 pages: Home, Calculator, and Quote.
- Added own styles to improve the look and feel.
- For every page, created a <Route>.
- Navigation links for all 3 routes are displayed on each page.
- Used `React Router` to make the routes work.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -25,41 +24,47 @@ const Calculator = () => {
...
   return (
-    <div className="calculator">
-      <div className="display">
-        {calculatorData.next !== null
-          ? calculatorData.next
-          : calculatorData.total || '0'}
+    <div className="calculator-container">
+      <div className="calculator-text">
+        <h1>Math Magicians</h1>
+        <h2>Let&apos;s do some math</h2>
```
Comment (Ibizugbe) [inline src/components/Calculator.js]: Nice job so far😉😉😉 everything looks good👍🏼. but let's try to make the nav bar section match the layout of the design😉. take note of the `math magician` header and the nav link and kindly ensure they are positioned in the right places as shown in the design. also, try to ensure your navbar occupies the full width of the page as shown in the design👍🏼.

design
image

yours
image

The logo should be placed on the nav bar and the navbar should take a 100% width to match the layout of the wireframe👍🏼.

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @Kaiserabbas 👋🏼👋🏼👋🏼, 

require_changes


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅✅

- [x] Linters are passing👍🏼.
- [x] `gitflow` is used👍🏼.
- [x] The app consists of 3 pages as required👍🏼.
- [x] Routes to each pages have been made available via react-router👍🏼.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (Kaiserabbas) [reply src/components/Calculator.js]: Thank you for for your kind review, I have addressed all the issues raised.

**Review by sumairq — APPROVED** [review]
### Hi @Kaiserabbas , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good job implementing changes
✔️Linters are passing
✔️Very professional README
✔️Correct Git flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**